src/main/java/com/iwaconsolti/school/miguel/config/SchoolConfig.java

```diff
@@ -0,0 +1,33 @@
+package com.iwaconsolti.school.miguel.config;
+
+import com.iwaconsolti.school.miguel.model.Courses;
+import com.iwaconsolti.school.miguel.model.School;
+import com.iwaconsolti.school.miguel.model.Students;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.Profile;
+
+@Configuration
+@Slf4j
+public class SchoolConfig {
+
+    @Profile("default")
+    @Bean
+    @Qualifier("GerardoInstitute")
+    public School gerardoInstitute(){
+        School school = new School();
+        school.setName("GerardoInstitute");
+        return new School();
```

Comment:
in here, you are setting the name as "Gerardoinstitute" but you are not using it because, i believe you wanted to return the school object with the name already setted as "gerardoinstitute".
```suggestion
        return school();
```

Reply:
I already confirmed what was mentioned and it was correct since a new object was returned

Reply:
I already confirmed what was mentioned and it was correct since a new object was returned